src/main/java/frc/robot/RobotContainer.java

```diff
@@ -121,7 +125,19 @@ private void configureButtonBindings() {
    */
   public Command getAutonomousCommand() {
```

Comment:
I'm not a fan of multiple return statements. It gets hard to follow. Instead, we should have a Command variable which is set on 129, 132, or 135, and then returned on 140